editreport_form.php

```diff
@@ -83,6 +87,30 @@ public function definition() {
         $mform->addHelpButton('remote', 'remote', 'block_configurable_reports');
         $mform->setDefault('remote', 0);
 
+        // Adds an embed link for easy copy/paste once the report is saved.
+        if (isset($this->_customdata['report']->id) && $this->_customdata['report']->id) {
+
+            $params = [
+                'id' => $this->_customdata['report']->id,
+                'courseid' => $this->_customdata['courseid'],
+                'embed' => true
+            ];
+            $url = new \moodle_url('/blocks/configurable_reports/viewreport.php', $params);
+
+            $mform->addElement('html',
```

Comment:
You may be able to use a static form element for free. This feels like somehting that should be in a renderer and called out to either way

Reply:
also put the link in a <code> or <pre> so its not a link, and instead you can triple click select it